src/helpers/render-document-file.js

```
@@ -458,6 +324,17 @@ async function findXMLEquivalent(docxDocumentInstance, vNode, xmlFragment, image
         console.log(`[DEBUG] findXMLEquivalent: buildImage returned null/undefined`);
       }
       return;
+    case 'svg':
+      const svgFragment = await buildSVGElement(docxDocumentInstance, vNode, null, imageOptions);
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)